x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/flattened/FlattenedFeatureSetUsage.java

```diff
@@ -17,6 +17,11 @@
 import java.io.IOException;
 import java.util.Objects;
 
+/**
+ * @deprecated This class exists for backwards compatibility with 7.12 only
+ * and should not be used for other purposes.
+ */
```

Comment:
Is this class for cases when `_xpack/usage` API is used in a mixed 7.12 and 7.13 nodes cluster, and this 7.13 node acts as a coordinating node and it needs to process the usage reply from 7.12 nodes?

Reply:
Yes exactly. We took the same approach for runtime fields -- @javanna explained it nicely in the `RuntimeFieldsFeatureSetUsage` Javadoc.